### ISSUE TITLE
Yaml-cpp error when running on Jetson TX2

### ISSUE THREAD

**Issue by laxnpander** [laxnpander/OpenREALM#40]
Error:
```
[2021-02-04 11:45:37.998] [info] config file loaded: 
terminate called after throwing an instance of 'YAML::TypedBadConversion<unsigned int>'
  what():  yaml-cpp: error at line 0, column 0: bad conversion
```

Seems to be coming from OpenVSLAM.

**Comment by laxnpander** [comment]
Okay, just a corrupted open vslam or open realm install. Fresh install solved the problem.

**Comment by laxnpander** [comment]
Weirdly, the problem has reoccured. It seems like there is something wrong in the yaml files. Likely some line ending, which is not visible but kills the parser.

**Comment by laxnpander** [comment]
Resolved by this: d3acfefd06c41f15209a547ede48bb992cb436d1
